roles/foreman_installer/defaults/main.yml

```
@@ -19,6 +19,10 @@ foreman_installer_module_prs_strategy: "merge"
 # Comma-separated list of "organization/module/branch", e.g. "theforeman/foreman/foo,theforeman/katello/bar"
 foreman_installer_module_branches: []
 
+foreman_installer_from_source: False
+installer_fork: theforeman
+installer_branch: develop
```

```suggestion
foreman_installer_git_fork: theforeman/foreman-installer
foreman_installer_git_branch: develop
```
Should we also support PR instead of branch like `module_pr.yml`?

---

Done.